packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1741,12 +1752,24 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlags.isSelected, value);
   }
 
+  /// Whether the owning [RenderObject] is currently enabled.
+  ///
+  /// A disabled object does not respond to user interactions. Only objects that
+  /// usually do respond to user interactions but currently don't (like a
```

Comment:
ultra-nit: what about 'usually respond to user interactions, but which currently do not (like a' to avoid the extra 'do'.